### ISSUE TITLE
Flaky test AsyncOffloadTest.testParallelStressMultipleThreadsSingleWorkerJitDisabled

### ISSUE THREAD

**Issue by ideoma** [questdb/questdb#2105]
### Describe the bug

```

java.lang.AssertionError: 
Expected :0
Actual   :1
<Click to see difference>


	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at io.questdb@6.3.1-SNAPSHOT/io.questdb.griffin.AsyncOffloadTest.lambda$testParallelStress$1(AsyncOffloadTest.java:249)
	at io.questdb@6.3.1-SNAPSHOT/io.questdb.test.tools.TestUtils.execute(TestUtils.java:657)
	at io.questdb@6.3.1-SNAPSHOT/io.questdb.griffin.AsyncOffloadTest.testParallelStress(AsyncOffloadTest.java:212)
	at io.questdb@6.3.1-SNAPSHOT/io.questdb.griffin.AsyncOffloadTest.testParallelStressMultipleThreadsSingleWorkerJitDisabled(AsyncOffloadTest.java:142)
```

### To reproduce

Run 30 seconds in a loop on ARM OSX

### Environment

```markdown
- **QuestDB version**: 6.3
- **OS**: OSX ARM
- **Browser**:
```

**Comment by puzpuzpuz** [comment]
@ideoma there should be some exceptions printed to stderr once the test fails. Could you share them?

**Comment by puzpuzpuz** [comment]
Should be fixed in #2107
